clients/js/package.json

```diff
@@ -64,15 +143,15 @@
     "node": ">=14.17.0"
   },
   "dependencies": {
-    "cliui": "^8.0.1",
     "isomorphic-fetch": "^3.0.0"
   },
   "peerDependencies": {
     "@google/generative-ai": "^0.1.1",
     "cohere-ai": "^5.0.0 || ^6.0.0 || ^7.0.0",
     "openai": "^3.0.0 || ^4.0.0",
     "voyageai": "^0.0.3-1",
-    "ollama": "^0.5.0"
+    "ollama": "^0.5.0",
+    "chromadb-default-embed": "^2.13.0"
   },
   "peerDependenciesMeta": {
```

Comment:
I think we need to mark chromadb-default-embed as optional in the peerDependenciesMeta

Reply:
👍 I didn't realize our getting started guides already recommended installing both `chromadb` and `chromadb-default-embed`